### PR TITLE
Match state info

### LINKED ISSUE #1250: [FEATURE] Update match state info
## 🚀 Feature
Update match state info according to the new design.

### PR DESCRIPTION
**Pull request by AngieDutra**
Closes #1250 and #1169

## Motivation

There is a gray bar always on display showing the metrics and a pink border bar displaying the match state info. Both needed style updates.

## Summary of changes

This PR changes:
- The top left bar was replaced for a darker bar with new icons
- The top middle bar was replaced for a top right only showing the fps and ping counters. The hashes are to be shown in a future update on settings. Their default state is "hidden". To see this top right bar go to `Settings > toggle metrics` and resume game

## Checklist
- [x] I have tested the changes locally.
- [x] I self-reviewed the changes on GitHub, line by line.
- [ ] Tests have been added/updated.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.
- [ ] I have tested the changes in another devices.
  - [ ] Tested in iOS.
  - [ ] Tested in Android.

### CODE REVIEW

**Review by ncontinanza — APPROVED** [review]
LGTM!

File: client/Assets/Images/UI/BattleUI/2.png
Comment (BertovDev) [inline client/Assets/Images/UI/BattleUI/2.png:1]: Maybe rename this?

File: client/Assets/Images/UI/BattleUI/kill-icon.png
Comment (BertovDev) [inline client/Assets/Images/UI/BattleUI/kill-icon.png:1]: With this we have 2 kill icons. Kill-icon and kills. I believe one should be removed.

File: client/Assets/Images/UI/BattleUI/time-icon.png
Comment (BertovDev) [inline client/Assets/Images/UI/BattleUI/time-icon.png:1]: Same with this. We have 2 time icons. (time-icon and time_new)

**Review by ncontinanza — APPROVED** [review]

**Review by ncontinanza — APPROVED** [review]

**Review by BertovDev — APPROVED** [review]
Works so fine! and I believe the whole ToggleButton script could be improved but that is something that was before this PR.

🚀🤙

**Pull request merged**